app/build.gradle

```diff
@@ -14,13 +14,18 @@ apply plugin: 'kotlin-android'
 ext {
     versionFile = new File(project.rootDir, 'app/src/main/assets/libopenmw/resources/version')
     calculateVersion = {
-        stream = new FileInputStream(versionFile)
-        return stream.readLines()[0]
+        if (versionFile.exists()) {
```

Comment:
Why's that needed?

Reply:
https://travis-ci.org/OpenMW/android-port/builds/524185792#L1566

Reply:
Was a fix from @terabyte so that travis-ci could build.